pkgs/top-level/python-packages.nix

```diff
@@ -4501,12 +4501,17 @@ in {
     cudaPackages = pkgs.cudaPackages_11_6;
   };
 
-  jaxlib-build = callPackage ../development/python-modules/jaxlib {
+  jaxlib-build = callPackage ../development/python-modules/jaxlib rec {
+    inherit (pkgs.darwin) cctools;
+    buildBazelPackage = pkgs.buildBazelPackage.override {
+      stdenv = pkgs.darwin.apple_sdk_11_0.stdenv;
+    };
```

Comment:
This seems like the kind of thing we should only do on Darwin?

Reply:
Yeah you're right, https://nixos.org/manual/nixpkgs/unstable/#sec-darwin documents that this is fine:
> On Linux, this will have the same effect as pkgs.callPackage, so you can use pkgs.darwin.apple_sdk_11_0.callPackage regardless of platform.

but it's unintuitive enough that I'll make this conditional.

Reply:
ah I see. yeah I think sticking to a conditional given the unintuitive-ness is a good idea